doc/IOTRACER.md

```diff
@@ -0,0 +1,304 @@
+271581184,nvme1n1,268439477,0,9,80578,160822252644,false,false,false,12,20540,183234040,16,Write,1
+...
+~~~
+
+And in case of JSON the output is:
+
+~~~{.sh}
+{"header":{"sid":"23","timestamp":"33493819292"},"io":{"lba":"1953524992","len":8,"ioClass":1,"operation":"Read","flush":false,"fua":false,"error":false,"latency":"144925","qd":"1"},"device":{"id":"271581185","name":"nvme0n1"}}
+{"header":{"sid":"25","timestamp":"33493993812"},"io":{"lba":"1953525152","len":8,"ioClass":1,"operation":"Read","flush":false,"fua":false,"error":false,"latency":"113330","qd":"1"},"device":{"id":"271581185","name":"nvme0n1"}}
+{"header":{"sid":"27","timestamp":"33494133732"},"io":{"lba":"0","len":8,"ioClass":1,"operation":"Read","flush":false,"fua":false,"error":false,"latency":"100530","qd":"1"},"device":{"id":"271581185","name":"nvme0n1"}}
+...
+{"header":{"sid":"18834","timestamp":"134120569337"},"io":{"lba":"1465145088","len":88,"ioClass":14,"operation":"Write","flush":false,"fua":false,"error":false,"latency":"32495","qd":"2"},"device":{"id":"271581185","name":"nvme0n1"},"file":{"id":"1610612801","offset":"112","size":"192"}}
+{"header":{"sid":"18838","timestamp":"134120638297"},"io":{"lba":"904","len":120,"ioClass":14,"operation":"Write","flush":false,"fua":false,"error":false,"latency":"40076","qd":"1"},"device":{"id":"271581185","name":"nvme0n1"},"file":{"id":"73","offset":"0","size":"203"}}
+{"header":{"sid":"18840","timestamp":"134120664253"},"io":{"lba":"1024","len":88,"ioClass":14,"operation":"Write","flush":false,"fua":false,"error":false,"latency":"30424","qd":"2"},"device":{"id":"271581185","name":"nvme0n1"},"file":{"id":"73","offset":"120","size":"203"}}
+{"header":{"sid":"18844","timestamp":"134120707517"},"io":{"lba":"488382416","len":48,"ioClass":14,"operation":"Write","flush":false,"fua":false,"error":false,"latency":"20330","qd":"1"},"device":{"id":"271581185","name":"nvme0n1"},"file":{"id":"536870985","offset":"0","size":"193"}}
+...
+~~~
+
+## What we collect, What we process?
+
+The bellow table contains telemetry content which is traces by iotrace. Also we
+included future plans for tracing.
+
+<style type="text/css">
+.tg  {border-collapse:collapse;border-spacing:0;border-color:#9ABAD9;}
+.tg td{font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:#9ABAD9;color:#444;background-color:#EBF5FF;}
+.tg th{font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;border-color:#9ABAD9;color:#fff;background-color:#409cff;}
+.tg .tg-j3py{background-color:#D2E4FC;border-color:#000000;text-align:left;vertical-align:top}
+.tg .tg-mcqj{font-weight:bold;border-color:#000000;text-align:left;vertical-align:top}
+.tg .tg-73oq{border-color:#000000;text-align:left;vertical-align:top}
```

Comment:
Please check if this shows up properly - it doesn't work in github diff (when switched to markdown viewer)